### PR TITLE
Renamed three files from example_processes

### LINKED ISSUE #423: Files in example_processes missing `.py` extension
The following Python files are missing the `.py` extension:

* launch_testing/example_processes/exit_code_proc
* launch_testing/example_processes/good_proc
* launch_testing/example_processes/terminating_proc

Since they are never invoked "naked" (e.g. always as `python3 /path/to/script`) there is no benefit to leaving off the `.py` extension to make them look like a native executable.

_Discovered in https://github.com/ament/ament_cmake/issues/213#issuecomment-631129869_

### PR DESCRIPTION
**Pull request by Khush-dev**
The following Python files were missing the .py extension:

- launch_testing/example_processes/exit_code_proc
- launch_testing/example_processes/good_proc
- launch_testing/example_processes/terminating_proc

Fixes #423 

PS: My first GitHub PR, any suggestions/tips are welcome

### CODE REVIEW

**Review by aprotyas — CHANGES_REQUESTED** [review]
Thanks for the PR!
You will have to change references or paths to these scripts as well, or else the tests referencing them will fail. Here's a list of the failed tests: https://build.ros2.org/job/Rpr__launch__ubuntu_focal_amd64/337/testReport/

For example, in `launch_testing/test_runner_results.py`, you need to change `exit_code_proc` to `exit_code_proc.py`.

https://github.com/ros2/launch/blob/b59bb6b88b40eaa82870542d93bdfb1a033d05c3/launch_testing/test/launch_testing/test_runner_results.py#L75-L79

Please do likewise for all references to the script and run tests (`colcon test --packages-select-regex launch`) before pushing the code!

**Comment by aprotyas** [comment]
CI:
Repos file: https://gist.github.com/aprotyas/d9c8fa4442a10b22e0868aa41e2a75f1
Build args: `--packages-up-to launch_testing`
Test args: `--packages-select launch_testing`
Job: https://ci.ros2.org/job/ci_launcher/9576/

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/15857/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/10534/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/13522/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/16152/)

File: launch_testing/example_processes/terminating_proc.py [hunk 1]
```diff
@@ -20,25 +20,25 @@
...
-        print("Emulating Work")
+        print('Emulating Work')
         time.sleep(1.0)
-        print("Done")
+        print('Done')
     except KeyboardInterrupt:
         pass
 
-    print("Shutting Down")
+    print('Shutting Down')
```
Comment (aprotyas) [inline launch_testing/example_processes/terminating_proc.py:42]: I understand the intention behind these changes, but these are out of the scope of this PR. It adds a lot of noise when reviewing, so I'd suggest not to make changes like this in the future.

Likewise for the same type of changes in the other scripts.

**Review by aprotyas — APPROVED** [review]
LGTM with green CI.

**Comment by aprotyas** [comment]
CI:
Repos file: https://gist.github.com/aprotyas/d9c8fa4442a10b22e0868aa41e2a75f1
Build args: `--packages-up-to launch_testing`
Test args: `--packages-select launch_testing`
Job: https://ci.ros2.org/job/ci_launcher/9576/

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/15857/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/10534/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/13522/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/16152/)

File: launch_testing/example_processes/terminating_proc.py (hunk 1 above)
Comment (Khush-dev) [reply launch_testing/example_processes/terminating_proc.py:42]: Alright, that makes sense. I'll keep that in mind.
Thank you for your valuable feedback! 😊️

**Review by hidmic — APPROVED** [review]

**Comment by hidmic** [comment]
Alright, going in. Thanks @Khush-dev !

**Pull request merged**